http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```
@@ -627,4 +635,40 @@ object Config {
 
   val leoPubsubMessageSubscriberConfig = config.as[LeoPubsubMessageSubscriberConfig]("pubsub.subscriber")
   val asyncTaskProcessorConfig = config.as[AsyncTaskProcessor.Config]("async-task-processor")
+
+  implicit private val secretKeyReader: ValueReader[SecretKey] =
+    stringValueReader.map(s => KubernetesName.withValidation(s, SecretKey).right.get)
```

can you throw an explicit exception here instead of doing `.get`